packages/web-app-files/src/composables/useFileListHeaderPosition/index.ts

```diff
@@ -0,0 +1,25 @@
+import { ref, nextTick, onMounted, Ref, watch } from '@vue/composition-api'
+import { useStore } from '../store'
+
+export const useFileListHeaderPosition = (): { y: Ref; refresh: () => void } => {
+  const store = useStore()
+  const y = ref(0)
+  const refresh = async (): Promise<void> => {
+    await nextTick()
+    const appBar = document.querySelector('#files-app-bar')
+    const height = appBar ? appBar.getBoundingClientRect().height : 0
+
+    if (y.value === height) {
+      return
+    }
+
+    y.value = height
+  }
+
+  // vue 3 (also 2 with composition api bridge) has no created hook (uses setup instead), so call it here
```

Comment:
I don't get your comment about the created hook. Is it meant to be a `FIXME: ` with an explanation why you're using `onMounted` instead of `onCreated`? Or something different?

Reply:
we had the listener on the created hook before, this doesn't exist anymore, the comment is just here if someone wonders. You're right it can go away